.github/workflows/release-openshift.yaml

```diff
@@ -0,0 +1,116 @@
+        uses: actions/checkout@v3.1.0
+        with:
+          repository: ${{ matrix.repository }}
+          ref: main
+          path: ${{ matrix.repository }}
+          token: ${{ github.token }}
+      - name: Prepare version
+        env:
+          VERSION: ${{ github.event.inputs.version }}
+          REPOSITORY: ${{ matrix.repository }}
+        run: |
+          cd $REPOSITORY       
+          mkdir -p "${REPO_PATH}/${VERSION}"
+
+          cd ../../mongodb-atlas-kubernetes
+          cp -r bundle.Dockerfile bundle/manifests bundle/metadata bundle/tests "../${REPOSITORY}/${REPO_PATH}/${VERSION}"
+      - name: Configure non-certified release
+        if: ${{ ! matrix.certified }}
+        env:
+          VERSION: ${{ github.event.inputs.version }}
+          REPOSITORY: ${{ matrix.repository }}
+        run: |
+          echo "Configure non-certified release"
+          cd "$REPOSITORY/$REPO_PATH"
+          sed -i.bak 's/COPY bundle\/manifests/COPY manifests/' "${VERSION}/bundle.Dockerfile"
+          sed -i.bak 's/COPY bundle\/metadata/COPY metadata/' "${VERSION}/bundle.Dockerfile"
+          sed -i.bak 's/COPY bundle\/tests\/scorecard/COPY tests\/scorecard/' "${VERSION}/bundle.Dockerfile"
+          rm "${VERSION}/bundle.Dockerfile.bak"
+      - name: Configure certified release
+        if: ${{ matrix.certified }}
+        env:
+          VERSION: ${{ github.event.inputs.version }}
+          REPOSITORY: ${{ matrix.repository }}
+          IMAGE: quay.io/mongodb/mongodb-atlas-kubernetes-operator
+          REGISTRY_TOKEN: ${{ secrets.REGISTRY_TOKEN }}
+          RH_CERTIFICATION_PYXIS_API_TOKEN: ${{ secrets.RH_CERTIFICATION_PYXIS_API_TOKEN }}
+          RH_CERTIFICATION_OSPID: ${{ secrets.RH_CERTIFICATION_OSPID }}
+        run: |
+          wget https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases/download/1.4.0/preflight-linux-amd64 -O preflight -q
+          chmod +x ./preflight && sudo mv ./preflight /usr/local/bin/preflight
+          
+          preflight --version
+          docker --version
+
+          docker login -u unused -p "${REGISTRY_TOKEN}" quay.io --authfile ./authfile.json
+          IMG_SHA=$("docker" inspect --format='{{ index .RepoDigests 0}}' "${IMAGE}":"${VERSION}")
+
+          # Do the preflight check first
+          preflight check container "${IMG_SHA}" --docker-config=./authfile.json
+
+          # Send results to RedHat if preflight finished without errors
+          preflight check container "${IMG_SHA}" \
+            --submit \
+            --pyxis-api-token="${RH_CERTIFICATION_PYXIS_API_TOKEN}" \
+            --certification-project-id="${RH_CERTIFICATION_OSPID}" \
+            --docker-config=./authfile.json
+
+          # Replace image version with SHA256
+          value="${IMG_SHA}" yq e -i '.spec.install.spec.deployments[0].spec.template.spec.containers[0].image = env(value)' \
+            "${VERSION}"/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml
+
```

Comment:
There is one more place to replace the image tag with image SHA: `.metadata.annotations.containerImage`

Reply:
Also, for certified operators please make sure that images have `quay.io/` prefix

Reply:
Done!
Also, as we now support multi-arch, we don't use the manifest digest(sha256) as the version anymore.
We just refer to the actual version of the release